src/runtime/client-config-manager.js

```diff
@@ -192,13 +192,23 @@ export class ClientConfigManager {
     const autoPromptConfigJson = json['autoPromptConfig'];
     let autoPromptConfig = undefined;
     if (autoPromptConfigJson) {
-      autoPromptConfig = new AutoPromptConfig(
-        autoPromptConfigJson.maxImpressionsPerWeek,
```

Comment:
Hmmmm doesn't this mean the response has to be changed and synchronized with this JS change? We may want to build the new config type with the old version of the autoPrompt JSON coming from the backend. Another major reason for this (at least in the short term) is that a breaking change in the JSON response will effect older versions of swg-basic

Reply:
Caught up offline. The new field is already being passed down on the backend so new JS should be fine to accept impression config

Reply:
The backend is still sending the maxImpressionsPerWeek field so publishers using the old version of swg-js can still consume this field. Looks like at least in the short term, it's not safe to remove that from the backend though.